vulnerabilities/import_runner.py

```diff
@@ -221,30 +313,31 @@ def _bulk_insert_impacted_and_resolved_packages(
             if not p:
                 p = _package_url_to_package(impacted_purl)
                 p.save()
-                impacted_packages[impacted_purl] = p
+                impacted_packages[impacted_purl] = p.id
 
-            ip = models.ImpactedPackage(
+            ip = models.PackageRelatedVulnerability(
                 vulnerability=vuln,
-                package=p,
+                package_id=p,
+                is_vulnerable=True
             )
-            impacted_refs.append(ip)
+            refs.append(ip)
 
         for resolved_purl in advisory.resolved_package_urls:
             # TODO Figure out when/how it happens that a package is missing from the dict and fix it
             p = resolved_packages.get(resolved_purl)
             if not p:
                 p = _package_url_to_package(resolved_purl)
                 p.save()
-                resolved_packages[resolved_purl] = p
+                resolved_packages[resolved_purl] = p.id
```

Comment:
What are the benefits of using ids vs objects there?

Reply:
As mentioned above this is : 

Overenginnering/pre-mature optimization . The idea was to use Dict[PackageURL, int] where the int is pk of Package , to save some memory because these dictionaries would grow big very fast. Carrying the Objects is way more expensive than pks so I had used it .

Reverting this.